public/index.js

```diff
@@ -1,34 +1,29 @@
 (function() {
 
-  // const message = document.getElementsByClassName('message')[0];
-  // const red = document.getElementsByClassName('red')[0];
-  // const blue = document.getElementsByClassName('blue')[0];
-  // const yellow = document.getElementsByClassName('yellow')[0];
-
+  const redBtn = document.getElementById('red-button');
```

Comment:
Remember that the class as a whole won't be familiar with ES6 syntax so you may need to explain it as you go if you decide to do that.